app/resources/v1/pacbio/pool_resource.rb

```diff
@@ -73,7 +76,11 @@ def updated_at
         @model.updated_at.to_fs(:us)
       end
 
-      def publish_messages
+      def publish_messages_on_creation
+        Emq::Publisher.publish(@model.primary_aliquot, Pipelines.pacbio, 'volume_tracking')
```

Comment:
Along with the primary_aliquot, I think, if  libraries are used while creating a pool, those aliquots from all libraries that are used in the pool also need to be sent for volume tracking.

Reply:
I am not sure, but also I am not sure what you by those aliquots (from all libraries that are used in the pool) are you referring  to the used_aliquots_attributes objects ?

Reply:
I mean  the `used_aliquots` within a pool which can be from a library or a request. As I mentioned, my idea is that the used_aliquots in a pool which is having source_type as library should also be published for volume_tracking, but good to confirm.

Reply:
We should send the primary aliquot when we create a pool as with libraries. We should then send the used aliquots when the run is created. From what I understand the libraries used aliquots are already sent. It might be worth having a quick meeting to go through this along with @BenTopping just to make sure we have everything covered.